src/Altinn.Profile.Integrations/Entities/RegistrySyncMetadata.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+
+namespace Altinn.Profile.Integrations.Entities
+{
+    /// <summary>
+    /// Table of metadata for last brreg kof sync batch
+    /// </summary>
+    [Table("registry_sync_metadata", Schema = "organization_notification_address")]
+    public class RegistrySyncMetadata
```

Comment:
Should we add a column with a known value so that we can more easily address the row in an update?

With the KRR batch metadata we're forced to perform a delete before doing an insert again. With a separate column with a fixed value we can simply just address the row in an update and update the two other columns.

Reply:
Why do we need a fixed value ? With only one entry in the table, it should be possible to just update that one :)

Reply:
Okay, let's try. Could be that the existing implementation has something with it that complicates the matter. I would also think it should be possible to update all (1) rows.